src/sentry/snuba/metrics.py

```diff
@@ -26,7 +26,7 @@
 from snuba_sdk.conditions import BooleanCondition
 from snuba_sdk.orderby import Direction, OrderBy
 
-from sentry.api.utils import get_date_range_from_params
+from sentry.api.utils import InvalidParams, get_date_range_from_params
```

Comment:
By importing `InvalidParams` from `utils` rather than from `sessions_v2`, we now automatically raise a 400 error for those here:

https://github.com/getsentry/sentry/blob/ef8eef59363229632e2ad5b2a5e8478c42b58d30/src/sentry/api/endpoints/organization_metrics.py#L45-L46